Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.10-slim as base
+FROM python:3.10 as base
```

Comment:
Just a follow-up on this. It does not seem to be possible to use off-hand (see [this CI run](https://github.com/SINTEF/entities-service/actions/runs/7726200383/job/21062009297)).
For now it doesn't matter much, as the docker image is currently only used for CI and local testing. If later we use it for proper deployment (which is should be), it will inevitably need to be updated any tricks to make `-slim` work can be investigated.